doc/PublicKeyCredentialSourceRepository.md

```diff
@@ -0,0 +1,196 @@
+Public Key Credential Source Repository
+=======================================
+
+The credential data is managed using a repository.
+The library does not provide any concrete implementation.
+It is up to you to create it depending on your application constraints.
+
+This repository must implement the interface `Webauthn\PublicKeyCredentialSourceRepository`.
+
+For example, you might save the data in the filesystem or in a database.
+
+# Filesystem Repository
+
+```php
+<?php
+/**
+ * EGroupware WebAuthn
+ *
+ * @link https://www.egroupware.org
+ * @author Ralf Becker <rb-At-egroupware.org>
+ * @package openid
+ * @license http://opensource.org/licenses/gpl-license.php GPL - GNU General Public License
+ */
+
+namespace EGroupware\WebAuthn;
+
```

Comment:
You probably ant the namespace and the @package openid above removed.
The class cluttered because it is a mixture of tab and space, that should be cleaned up too.